dockerfile_scripts/scrape_libs.sh

```diff
@@ -41,10 +41,10 @@ then
    export CUDA_CACHE_PATH=${CUDA_CACHE_PATH:=${tmp_nvcache_dir}}
 
    export TF_FORCE_GPU_ALLOW_GROWTH=${TF_FORCE_GPU_ALLOW_GROWTH:=true}
-
    # NOTE: Disable memory registration to workaround the current issues
    #       between libfabric and cuda.  When those issus are resolved,
    #       simply set the vaiable to 0 before launching the container.
+   export FI_MR_CACHE_MONITOR=userfaultfd
```

Comment:
Should we add a comment here on what this variable does/why it's important ?

Reply:
Yeah I can make that change.

Reply:
Comments have been added!